src/main/java/org/groomUniv/meet/common/security/JwtTokenProvider.java

```diff
@@ -0,0 +1,126 @@
+package org.groomUniv.meet.common.security;
+
+import io.jsonwebtoken.*;
+import io.jsonwebtoken.io.Decoders;
+import io.jsonwebtoken.security.Keys;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.security.core.Authentication;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.security.authentication.UsernamePasswordAuthenticationToken;
+import org.springframework.security.core.GrantedAuthority;
+import org.springframework.security.core.authority.SimpleGrantedAuthority;
+import org.springframework.security.core.userdetails.User;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.stereotype.Component;
+
+import java.security.Key;
+import java.util.*;
+import java.util.stream.Collectors;
+
+@Slf4j
+@Component
+public class JwtTokenProvider {
+    private static final long ACCESS_TOKEN_EXPIRE_TIME = 60 * 60 * 1000L;       // 1시간
+    private static final long REFRESH_TOKEN_EXPIRE_TIME = 7 * 24 * 60 * 60 * 1000L; // 1주일
+    long now = (new Date()).getTime();
+    private final Key key;
+
+
+    //application.yml에 저장한 secret값을 가져와서 key에 저장하기
+    public JwtTokenProvider(@Value("${jwt.secret}") String secretKey) {
+        byte[] keyBytes = Decoders.BASE64.decode(secretKey);    // secretKey (Base64로 인코딩된 문자열)를 디코딩하여 byte 배열로 변환
+        this.key = Keys.hmacShaKeyFor(keyBytes);    // 디코딩된 byte 배열을 사용하여 HMAC-SHA 알고리즘에 맞는 암호화 키 생성
+    }
```

Comment:
저번에 말씀드린 것 같은데,
이런 방향이면 더 좋을 것 같아요
```suggestion
    @Value("${jwt.token.access-expiration-time}")
    private long accessTokenExpirationTime;

    @Value("${jwt.token.refresh-expiration-time}")
    private long refreshTokenExpirationTime;

    @PostConstruct
    protected void init() {
        byte[] keyBytes = Decoders.BASE64.decode(secretKey);
        key = Keys.hmacShaKeyFor(secretKeyBytes);
    }
```

Reply:
감사합니다! 혹시 파라미터에 바로 넣지 않는 이유가 있나요..?!

Reply:
가독성때문일 것 같아요

Reply:
아하 감사합니다! ㅎㅎ